Routes/index.js

```diff
@@ -0,0 +1,28 @@
+//Seccion 1. importaciones
+
+const express  =require("express")
+const res = require("express/lib/response")
+const router = express.Router()
+
+//Seccion 2. Rute
+
+router.get("/", (req,res)=> {
+
+    res.render("Hola de nuevo")
+})
+
+router.get ("/index", (req,res) => {
```

Comment:
Te recomiendo tener runa ruta raiz que renderice un hbs como la siguente ya que en todos los sitios siempre hay una vista que se renderiza en la raiz del server```suggestion
router.get ("/", (req,res) => {
```